manager/state/raft/raft.go

```diff
@@ -40,6 +42,14 @@ import (
 )
 
 var (
+	// EnvRaftHeartbeatTick is the env to set to customize the Raft heartbeat tick
+	EnvRaftHeartbeatTick = "SWARMKIT_RAFT_HEARTBEAT_TICK"
+
+	// EnvElectionTick is the env to set to customize the Raft leader election delay
```

Comment:
Circleci reports that this comment is missing a `Raft` in `EnvRaftElectionTick` and is failing because of it.